pandas_gpt/__init__.py

```diff
@@ -1,103 +1,191 @@
+from dataclasses import dataclass
+from typing import Any, Callable
 import pandas as pd
-
-verbose = False # Override default setting with `pandas_gpt.verbose = True`
-mutable = False # Override default setting with `pandas_gpt.mutable = True`
-
-model = 'gpt-3.5-turbo'
-completion_config = {}
-
-template = '''
+import os
+
+__all__ = [
+    "verbose",
+    "mutable",
+    "completer",
+    "system_prompt",
+    "template",
+    "Ask",
+    "AskAccessor",
+    "OpenAI",
+    "OpenRouter",
+    "LiteLLM",
+]
+
+
+@dataclass
+class OpenAI:
```

Comment:
Question: Does it make sense to move various completers to their own modules or do you want all the examples that are packaged with pandas-gpt to be in `__init__.py`?

Reply:
Thanks for the changes, I think the new structure is great. I can add Gemma related changes after you merge this.